// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/es6/functionDeclarations/FunctionDeclaration8_es6.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -4,19 +4,15 @@
 declare var v: invalid;
 
 /// [Errors] ////
 
-FunctionDeclaration8_es6.ts(1,11): error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
 FunctionDeclaration8_es6.ts(1,12): error TS2304: Cannot find name 'yield'.
 FunctionDeclaration8_es6.ts(1,20): error TS2304: Cannot find name 'foo'.
 FunctionDeclaration8_es6.ts(1,20): error TS9013: Expression type can't be inferred with --isolatedDeclarations.
 
 
-==== FunctionDeclaration8_es6.ts (4 errors) ====
+==== FunctionDeclaration8_es6.ts (3 errors) ====
     var v = { [yield]: foo }
-              ~~~~~~~
-!!! error TS9014: Computed properties must be number or string literals, variables or dotted expressions with --isolatedDeclarations.
-!!! related TS9027 FunctionDeclaration8_es6.ts:1:5: Add a type annotation to the variable v.
                ~~~~~
 !!! error TS2304: Cannot find name 'yield'.
                        ~~~
 !!! error TS2304: Cannot find name 'foo'.
